include/zephyr/posix/signal.h

```diff
@@ -12,6 +12,60 @@
 extern "C" {
 #endif
 
+enum {
+	SIGABRT = 1,
+	SIGALRM,
+	SIGBUS,
+	SIGCHLD,
+	SIGCONT,
+	SIGFPE,
+	SIGHUP,
+	SIGILL,
+	SIGINT,
+	SIGKILL,
+	SIGPIPE,
+	SIGQUIT,
+	SIGSEGV,
+	SIGSTOP,
+	SIGTERM,
+	SIGTSTP,
+	SIGTTIN,
+	SIGTTOU,
+	SIGUSR1,
+	SIGUSR2,
+	SIGPOLL,
+	SIGPROF,
+	SIGSYS,
+	SIGTRAP,
+	SIGURG,
+	SIGVTALRM,
+	SIGXCPU,
+	SIGXFSZ,
+};
+
+/* Signal set management definitions and macros. */
+#define MIN_SIGNO       1                       /* Lowest valid signal number */
```

Comment:
I might move some of these into the enum above

Reply:
I'm somewhat concerned that using an ENUM will allow pre-processor defines to override these without generating a warning.

Reply:
Maybe I shouldn't be lazy and should have used defines instead..

Reply:
I think technically you can do both 😁 but some people might find that a bit too verbose

Reply:
Generally though, `#define`s would give you more compile-time constness which is A Good Thing™️ for size.